### PR TITLE
[WIP] 452 - restrict uploads to mp4 (and add some messaging)

### PR DESCRIPTION
**Pull request by bent0b0x**
**Issue**
#452 

**Work Done**
- restrict file input to only accept `.mp4` by default
- add messaging to indicate that only `.mp4`s are accepted

**Known limitations**
- users can still drag non-mp4 files into the dropzone. But at least we are warning them now that this will fail. Detecting the file type of a dragged file is possible but not sure if it is worth the time right now. And it would likely implore us a dependency which I would like to avoid until we get the bundle to a more manageable size. In addition, I believe it would require us to convert the entire file into a `Uint8Array` (which would be a performance concern depending on file size).

**Screenshots**

image

image

other file types are not selectable
image

### CODE REVIEW

**Comment by bent0b0x** [comment]
@ya7ya is mp4 still the only supported file type?

**Comment by bent0b0x** [comment]
@pedrocasa @felipegaucho what do you think? I'm happy to update the messaging, both the words and where they are located

File: src/scripts/components/Dropzone.js [hunk 1]
```diff
@@ -1,657 +0,0 @@
...
-import React from 'react'
-import PropTypes from 'prop-types'
-import styled from 'styled-components'
-
-const DropzoneWrapper = styled.div`
-  cursor: pointer;
-  width: 200px;
-  height: 200px;
-  border: 2px dashed #ccc;
-  background: #666;
```
Comment (bent0b0x) [inline src/scripts/components/Dropzone.js]: Unused file 😄

File: src/scripts/components/widgets/FilesUploader.js [hunk 2]
```diff
@@ -147,9 +146,36 @@ class FilesUploader extends Component<Props, Object> {
...
+          onClick={this.onCheck}
+          onChange={this.onFileChosen}
+          onDragEnd={this.onDrag}
+        />
+        <UploadCover>
+          {card && <Title>Upload video</Title>}
+          <UploadCoverIcon>
+            {card && (
+              <Icon>
+                <FilesUploaderSvg />
```
Comment (bent0b0x) [inline src/scripts/components/widgets/FilesUploader.js]: Refactored some duplicate code

**Comment by pedrocasa** [comment]
Amazing work Ben, maybe I would change the color and add the (+) icon

**Comment by bent0b0x** [comment]
Sounds good. @pedrocasa what color? I'll leave the plus icon out for this one as that is being handled by a separate issue

**Comment by pedrocasa** [comment]
let's use this #65667C

**Comment by jellegerbrandy** [comment]
marked this pr as [wip] so we won't merge it accidentally, let me know when it's done

**Pull request merged**